# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: netwerk/base/FuzzySecurityInfo.h
# Commit: 23b864e14db0
# Full Hash: 23b864e14db0cf874d0e126b6299c8c44922dde3
# Author: Dana Keeler <dkeeler@mozilla.com>
# Date: 2022-11-16 09:31:06
# Regressor Bug: 1791633
# File Overlap Count: 1
# Description:
#   Bug 1791633 - separate nsITLSSocketControl from nsITransportSecurityInfo r=necko-reviewers,kershaw,jschanck
#   
#   Depends on D160311
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D160313
# ==============================================================================

diff -r 0bcba3375ec0 -r 23b864e14db0 netwerk/base/FuzzySecurityInfo.h
--- a/netwerk/base/FuzzySecurityInfo.h	Tue Nov 15 20:03:29 2022 +0000
+++ /dev/null	Thu Jan 01 00:00:00 1970 +0000
@@ -1,43 +0,0 @@
-/* -*- Mode: C++; tab-width: 2; indent-tabs-mode: nil; c-basic-offset: 2 -*- */
-/* vim:set ts=2 sw=2 sts=2 et cindent: */
-/* This Source Code Form is subject to the terms of the Mozilla Public
- * License, v. 2.0. If a copy of the MPL was not distributed with this
- * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
-
-#ifndef FuzzySecurityInfo_h__
-#define FuzzySecurityInfo_h__
-
-#include "nsCOMPtr.h"
-#include "nsIInterfaceRequestor.h"
-#include "nsITLSSocketControl.h"
-#include "nsITransportSecurityInfo.h"
-#include "prerror.h"
-#include "sslproto.h"
-#include "sslt.h"
-
-namespace mozilla {
-namespace net {
-
-class FuzzySecurityInfo final : public nsITransportSecurityInfo,
-                                public nsIInterfaceRequestor,
-                                public nsITLSSocketControl {
- public:
-  FuzzySecurityInfo();
-
-  NS_DECL_THREADSAFE_ISUPPORTS
-  NS_DECL_NSITRANSPORTSECURITYINFO
-  NS_DECL_NSIINTERFACEREQUESTOR
-  NS_DECL_NSISSLSOCKETCONTROL
-
- protected:
-  virtual ~FuzzySecurityInfo();
-
- private:
-  nsCOMPtr<nsIInterfaceRequestor> mCallbacks;
-
-};  // class FuzzySecurityInfo
-
-}  // namespace net
-}  // namespace mozilla
-
-#endif  // FuzzySecurityInfo_h__